packages/fast-components-react-base/app/index.tsx

```diff
@@ -23,7 +25,10 @@ const formChildOptions: IFormChildOption[] = formChildFromExamplesFactory(exampl
 /* tslint:disable */
 function render(): void {
     ReactDOM.render(
-        <Site title={"FAST base component documentation"} formChildOptions={formChildOptions}>
+        <Site formChildOptions={formChildOptions}>
+            <SiteTitle slot={"title"}>
```

Comment:
Is there an enum for slots? If not I can file a followup task

Reply:
no enum, file away!